### PR TITLE
Update README.md with gms myeloid5.1

### PR DESCRIPTION
**Pull request by annaengstrom**
Added gmsmyeloid5.1 to README. 

Check list if changes affect bed files:

- [x] Update shasum
- [x] Update annotations
- [x] Update filename if relevant
- [x] Update person who designed/changed it
- [x] Update bed file version
- [ ] Update UCSC track
- [x] Update statusDB
- [ ] ~Deploy on Hasta~ **not applicable**
- [x] Update LIMS

### CODE REVIEW

**Comment by annaengstrom** [comment]
Please update panel size.

File: README.md [hunk 1]
```diff
@@ -82,6 +83,25 @@ Repository to collect and share bed files for each bait set. This repository doe
...
+##### gmsmyeloid5.1: <a name="CG5.1"></a>
+
+| Feature type | Count |
+| --- | --- |
+| Genes | 000 |
+| Unique Transcripts | 000 |
+| Transcripts with all exons covered | 000 |
+| Transcripts with at least one exon covered | 000 |
+| Design Padding | 000bp |
+| Panel size |  bp |
```
Comment (hassanfa) [inline README.md:95]:
```suggestion
| Panel size |  712494 bp |
```

**Comment by hassanfa** [comment]
I updated panel size. Now need to add gms_myeloid_5.1 to https://clinical-api.scilifelab.se/admin/bedversion/ as well.

Both gms_myeloid_5.1 and gms_myeloid_5.2 (and all others) will point to the same name BED: GMSmyeloid. Is this intended? https://clinical-api.scilifelab.se/admin/bed/ 

I guess only short name is now on LIMS right?

**Comment by hassanfa** [comment]
bump! @annaengstrom It'd be great to finish this and close it.

**Comment by annaengstrom** [comment]
> GMSmyeloid

It is intended that all versions x.1 and x.2 will point to the same [Bed](https://clinical-api.scilifelab.se/admin/bed/).

**Comment by hassanfa** [comment]
That can't be right. GMS 5.1 is different bedfiles than 5.2, and they should point to different bedfiles.

**Comment by annaengstrom** [comment]
LIMS and statusDB are now updated.

Screenshot 2020-04-17 at 16 10 35
Screenshot 2020-04-17 at 16 11 34

**Comment by hassanfa** [comment]
@annaengstrom wait!

**Comment by annaengstrom** [comment]
> That can't be right. GMS 5.1 is different bedfiles than 5.2, and they should point to different bedfiles.

They point to different bed files (according to short name), but the same Bed. See AM doc 2070 - I can also explain more next week.

**Comment by hassanfa** [comment]
GMS myeloid 5.1 and 5.2 can not point to same bed file 😣

**Comment by annaengstrom** [comment]
They point to different bed files

**Comment by annaengstrom** [comment]
Bed and Bed version in statusDB are very different

**Comment by hassanfa** [comment]
i see. my bad, it does take bed_version.  Good news everyone! https://www.youtube.com/watch?v=g8IVI0sZ6F8

https://github.com/Clinical-Genomics/cg/blob/d19f10c4e50d65c08717f08f8c0776933b33f1c4/cg/meta/workflow/base.py#L13

**Comment by hassanfa** [comment]
I'll bump version right away! Thanks for fixing this fast.

**Pull request merged**